doc/python/marker-style.md

```diff
@@ -305,6 +305,111 @@ fig.show()
 
 ```
 
+### Custom Marker Symbols
+
+The `marker_symbol` attribute allows you to customize the symbol used to represent markers. 
+
+In the following figure, hover over a symbol to see its name. Set the `marker_symbol` attribute equal to that name to change the marker symbol in your figure.
```

Comment:
How about making a first plot where only symbols < 100 are displayed, then explain what the figure reference says "Adding 100 is equivalent to appending "-open" to a symbol name. Adding 200 is equivalent to appending "-dot" to a symbol name. Adding 300 is equivalent to appending "-open-dot" or "dot-open" to a symbol name." and make a second (more busy) chart with all symbols?